src/libstore/store-api.hh

```diff
@@ -261,10 +262,40 @@ public:
 
 protected:
 
+    struct PathInfoCacheValue {
+
+        PathInfoCacheValue(std::shared_ptr<const ValidPathInfo> value) :
+            time_point(std::chrono::steady_clock::now()),
+            value(value) {}
```

Comment:
Since we're using C++20 now, constructors for "trivial" data classes can be omitted using designated initializers, e.g.
```
PathInfoCacheValue { .value = some_value }
```
and `time_point` could have a default initializer:
```
         std::chrono::time_point<std::chrono::steady_clock> time_point = std::chrono::steady_clock::now();
```